### PR TITLE
Fix bug FindFirst and update PostgreSQL tests

### PR DESCRIPTION
**Pull request by jenovateurs**
Hello team!

A lot of things in this PR : 
- I updated some tests to handle PostgreSQL.
- I updated some Migration classes to create tables in the PostgreSQL database.
- I found one bug with the function getPhql and I hope I have fixed it correctly.

I discovered the bug by doing the tests 
`mvcModelFindFirstStringPrimaryKey (database/Mvc/Model/FindFirstCest.php).`

The error was triggered with PostgreSQL.
When we do a:
`ModelWithStringPrimary::findFirst('135');`

The primary key of ModelWithStringPrimary is a string.
And the function findFirst writes the condition like that:
`... where uuid = 135`

PostgreSQL triggers an error like character varying = integer (there is an error...).

So I updated getPhSQL and checked if the type of the simple condition is a string, I put single quotes around the condition.

The updated file is phalcon/Mvc/Model/Query/Builder.zep

Thanks for your review/help 🧙.

* Type: bug fix | code quality
* Link to issue: #14874

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [x] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Thanks

### CODE REVIEW

File: ext/kernel/filter.c [hunk 1]
```diff
@@ -218,7 +218,7 @@ void zephir_escape_multi(zval *return_value, zval *param, const char *escape_cha
 		/**
 		 * Alphanumeric characters are not escaped
 		 */
-		if (value < 123 && isalnum(value)) {
+		if (value < 256 && isalnum(value)) {
```
Comment (niden) [inline ext/kernel/filter.c]: @ruudboon Is this correct, talking about the work you did with multibyte on macOS

File: ext/php_phalcon.h [hunk 2]
```diff
@@ -14,7 +14,7 @@
 #define PHP_PHALCON_VERSION     "4.0.5"
 #define PHP_PHALCON_EXTNAME     "phalcon"
 #define PHP_PHALCON_AUTHOR      "Phalcon Team and contributors"
-#define PHP_PHALCON_ZEPVERSION  "0.12.17-6724dbf"
+#define PHP_PHALCON_ZEPVERSION  "0.12.16-12256a1"
```
Comment (niden) [inline ext/php_phalcon.h]: I think this is wrong since Zephir is at 0.12.17. We might have to regenerate the ext folder

File: CHANGELOG-4.0.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+# [4.0.6](https://github.com/phalcon/cphalcon/releases/tag/v4.0.5) (2020-xx-xx)
+## Added
+
+## Changed
+
+## Fixed
+- Fixed `Phalcon\Mvc\Model\Query\Builder::getPhql` to add single quote between string value on a simple condition. [#14874](https://github.com/phalcon/cphalcon/issues/14874)
```
Comment (niden) [inline CHANGELOG-4.0.md]: You need to put your name at the end of the line (with github link) to get credit for this :)

File: tests/database/Db/Adapter/Pdo/DescribeColumnsCest.php [hunk 4]
```diff
@@ -56,4 +56,29 @@ public function dbAdapterPdoDescribeColumnsOnUpdate(DatabaseTester $I)
         $I->assertSame('CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP', $columns[2]->getDefault());
         $I->assertSame('NULL on update CURRENT_TIMESTAMP', $columns[3]->getDefault());
     }
+
+    /**
+     * Tests Phalcon\Db\Adapter\Pdo :: describeColumns()
+     *
+     * @author Jeremy PASTOURET <https://github.com/jenovateurs>
+     * @since  2019-12-07
```
Comment (niden) [inline tests/database/Db/Adapter/Pdo/DescribeColumnsCest.php]: Date is wrong :)

**Review by niden — CHANGES_REQUESTED** [review]
Excellent work. Thank you Jeremy. Some minor comments and a question for @ruudboon 

Other than that this is solid work!

File: ext/kernel/filter.c (hunk 1 above)
Comment (jenovateurs) [reply ext/kernel/filter.c]: I didn't change manually this file, it happends when I compile Phalcon with Zephir.
I will restore the file like it is in the v4.0 branch.

File: ext/php_phalcon.h (hunk 2 above)
Comment (jenovateurs) [reply ext/php_phalcon.h]: Same things that previously

File: CHANGELOG-4.0.md (hunk 3 above)
Comment (jenovateurs) [reply CHANGELOG-4.0.md]: I will do it.

File: tests/database/Db/Adapter/Pdo/DescribeColumnsCest.php (hunk 4 above)
Comment (jenovateurs) [reply tests/database/Db/Adapter/Pdo/DescribeColumnsCest.php]: You're right, thanks for your comments and for your review ;-).

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @jenovateurs

**Pull request merged**